src/benchmarklib/benchmark_sql_executor.cpp

```diff
@@ -82,7 +82,7 @@ BenchmarkSQLExecutor::~BenchmarkSQLExecutor() {
 
 void BenchmarkSQLExecutor::commit() {
   Assert(transaction_context && !transaction_context->is_auto_commit(),
-         "Can only explicitly commit transaction if auto-commit is disabled");
+         "Can only explicitly commit transaction if auto-commit is disabled.");
```

Comment:
Minor: Can re use another word than `disabled`?

Reply:
`"Can only explicitly commit transaction if auto-commit is set to 'No'."`
`"Can only explicitly commit transaction if auto-commit is off."`

What would you prefer?